code/modules/cargo/packs/research.dm

```diff
@@ -0,0 +1,77 @@
+/datum/supply_pack/science
+	group = "Science"
+	crate_type = /obj/structure/closet/crate/science
+
+/datum/supply_pack/science/faction_tech/nanotrasen
+	name = "NanoTrasen Equipment Design Drive"
+	desc = "Contains one NanoTrasen Equipment Design Drive."
+	cost = 1000
+	small_item = TRUE
+	faction = /datum/faction/nt
+	contains = list(/obj/item/faction_tech/nanotrasen)
+
+/datum/supply_pack/science/faction_tech/hardliners
+	name = "Hardliners Equipment Design Drive"
+	desc = "Contains one Hardliners Equipment Design Drive."
+	cost = 1000
+	small_item = TRUE
+	faction = /datum/faction/syndicate/hardliners
+	contains = list(/obj/item/faction_tech/hardliners, /obj/item/faction_tech/scarborough)
+
+/datum/supply_pack/science/faction_tech/gorlex_republic
+	name = "New Gorlex Republic Equipment Design Drive"
+	desc = "Contains one New Gorlex Republic Equipment Design Drive."
+	cost = 1000
+	small_item = TRUE
+	faction = /datum/faction/syndicate/ngr
+	contains = list(/obj/item/faction_tech/gorlex_republic, /obj/item/faction_tech/scarborough)
+
+/datum/supply_pack/science/faction_tech/cybersun
+	name = "Cybersun Equipment Design Drive"
+	desc = "Contains one Cybersun Equipment Design Drive."
+	cost = 1000
+	small_item = TRUE
+	faction = /datum/faction/syndicate/cybersun
+	contains = list(/obj/item/faction_tech/cybersun, /obj/item/faction_tech/scarborough)
+
+/datum/supply_pack/science/faction_tech/inteq
+	name = "Inteq Risk Management Group Equipment Design Drive"
+	desc = "Contains one Inteq Risk Management Group Equipment Design Drive."
+	cost = 1000
+	small_item = TRUE
+	faction = /datum/faction/inteq
+	contains = list(/obj/item/faction_tech/inteq)
+
+/datum/supply_pack/science/faction_tech/clip
+	name = "Confederated League of Independent Planets Equipment Equipment Design Drive"
+	desc = "Contains one Confederated League of Independent Planets Equipment Equipment Design Drive."
+	cost = 1000
+	small_item = TRUE
+	faction = /datum/faction/clip
+	contains = list(/obj/item/faction_tech/clip)
+
+/*
+/datum/supply_pack/science/faction_tech/pirate
```

Comment:
There's frontiersmen ships that could probably use this? or independent ships if we really wanna get freaky since 'pirate tech' is just unlicensed guns